apps/dashboard/app/models/script.rb

```diff
@@ -151,9 +163,13 @@ def submit(options)
     adapter = adapter(options[:auto_batch_clusters]).job_adapter
     render_format = adapter.class.name.split('::').last.downcase
 
+
+    #submit_options = submit_opts(options, render_format)
+    #submit_options[:output_path] = "output.txt";
+
     job_script = OodCore::Job::Script.new(**submit_opts(options, render_format))
 
-    job_id = Dir.chdir(project_dir) do
+    job_id = Dir.chdir(output_dir) do
```

Comment:
I don't think this should change. This sets the `CWD` of the job and users will need to rely on the fact that the job starts in the project's directory.

Reply:
ok - will revert